src/current/v23.1/changefeed-messages.md

```diff
@@ -96,6 +97,12 @@ The message envelope contains a primary key array when your changefeed is emitti
         {"after": {"city": "washington dc", "creation_time": "2019-01-02T03:04:05", "current_location": "46227 Jeremy Haven Suite 92", "ext": {"brand": "Schwinn", "color": "red"}, "id": "298cc7a0-de6b-4659-ae57-eaa2de9d99c3", "owner_id": "beda1202-63f7-41d2-aa35-ee3a835679d1", "status": "in_use", "type": "bike"}, "key": ["washington dc", "298cc7a0-de6b-4659-ae57-eaa2de9d99c3"]}
         ~~~
 
+To include a `before` field that contains the state of a row before an update in the changefeed message, use the `diff` option with `wrapped`:
+
+~~~
+{"after": {"city": "seattle", "end_address": null, "end_time": null, "id": "f6c02fe0-a4e0-476d-a3b7-91934d15dce2", "revenue": 25.00, "rider_id": "14067022-6e9b-427b-bd74-5ef48e93da1f", "start_address": "2 Michael Field", "start_time": "2023-06-02T15:14:20.790155", "vehicle_city": "seattle", "vehicle_id": "55555555-5555-4400-8000-000000000005"}, "before": {"city": "seattle", "end_address": null, "end_time": null, "id": "f6c02fe0-a4e0-476d-a3b7-91934d15dce2", "revenue": 25.00, "rider_id": "14067022-6e9b-427b-bd74-5ef48e93da1f", "start_address": "5 Michael Field", "start_time": "2023-06-02T15:14:20.790155", "vehicle_city": "seattle", "vehicle_id": "55555555-5555-4400-8000-000000000005"}, "key": ["seattle", "f6c02fe0-a4e0-476d-a3b7-91934d15dce2"]}
```

Comment:
This is not content I'm familiar with, but I cannot spot any `diff` or `wrapped` options here.

Reply:
Hey @mdlinville, so the `before` field is in the output. This section was mostly concerned with output and adapting the envelope of the message to different formats / using different metadata fields.

Having said that, I can see why that was confusing and have added the changefeed creation statements throughout the section to clear up any confusion.